.github/workflows/containerize-agent.yaml

```diff
@@ -129,6 +133,7 @@ jobs:
           tags: |
             greatexpectations/agent:${{ steps.get_version.outputs.POETRY_VERSION }}
             greatexpectations/agent:latest
+            greatexpectations/agent:stable
```

Comment:
Is there ever going to be a time when the `latest` and `stable` images are not the same?

Reply:
No, not to my knowledge--with these changes, `stable` and `latest` would always be the same.

Reply:
(The thinking here was to keep `latest` just in case users try to pull an image without specifying a tag, since `docker pull` defaults to using the `latest` tag if another tag isn't specified.

Reply:
That makes sense, is the stable tag present to give peace of mind then to our users?

Reply:
Yeah pretty much - we may decide to decouple `stable` and `latest` in the future, but that isn't being considered in this epic. The main change in this epic is that we'll start using pre-release versioning and will only push release version images to dockerhub once/week. (Add'l info here: https://greatexpectations.atlassian.net/wiki/spaces/SUP/pages/1019838486/Agent+Versioning+Release+Strategy)